### PR TITLE
base: include git-core

### PR DESCRIPTION
**Pull request by dustymabe**
Useful for transferring files or doing container builds via:

```
podman build -t myimg git://mygitforge.com/bob/example.git
```

### CODE REVIEW

**Comment by jlebon** [comment]
https://github.com/coreos/fedora-coreos-config#adding-packages-to-the-os  :)

Otherwise, LGTM!

**Comment by dustymabe** [comment]
>  :)

:) - thanks! updated

**Comment by cgwalters** [comment]
Also xref https://bugzilla.redhat.com/show_bug.cgi?id=1161251

**Comment by dustymabe** [comment]
> Also xref https://bugzilla.redhat.com/show_bug.cgi?id=1161251

Thanks for the link. There's no action I need to take, correct?

**Review by jlebon — APPROVED** [review]

**Comment by dustymabe** [comment]
ok based on some other feedback I created a new manifest in a second commit called `moving-files-around.yaml` that can be inherited by RHCOS.

**Review by miabbott — APPROVED** [review]
LGTM!

File: moving-files-around.yaml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+# Moving files around and verifying them
```
Comment (jlebon) [inline moving-files-around.yaml]: How about `file-transfer.yaml` for this file?

**Review by jlebon — APPROVED** [review]

**Pull request merged**